### PR TITLE
enhance documentation

### PR DESCRIPTION
**Pull request by mcm1957**
Add sections describing mentainence steps to add / modify / delete translations for developers

### CODE REVIEW

**Comment by mcm1957** [comment]
readme.md
add sections describing howto add / modify / delete a translation

admin.md
add plugin "Cleanup translation files"

**Comment by mcm1957** [comment]
@UncleSamSwiss 

Hi, könntest du wenn du Zeit und Lust hast den PR mal überfliegen und mergen oder m ir Bescheid geben was geändert werden sollte?

File: translate.md
Comment (UncleSamSwiss) [inline translate.md:1]: I disagree with that: the dash character is only used seldom in the English language. We may write "IP Address" as casing is dependent on the context, but I haven't seen "IP-Address" in English text. Ref: https://en.wikipedia.org/wiki/IP_address

**Review by UncleSamSwiss — CHANGES_REQUESTED** [review]
Thanks a lot for the good additions to the texts. There is only one point where I disagree.

File: translate.md
Comment (mcm1957) [reply translate.md:1]: I reverted this change.
IP address stays unchanged

**Comment by mcm1957** [comment]
IP-Address changed back to IP address

@UncleSamSwiss 
Thanks for reviewing.
Please let me know whether I have to do something to get the status switched back again (is still at "Changes requested"). Don't see a buton for "Recheck or something like this.

**Review by UncleSamSwiss — APPROVED** [review]

**Pull request merged**